class-tgm-plugin-activation.php

```diff
@@ -2797,17 +2840,18 @@ public function prepare_items() {
 		}
 
 
-		// *********** DEPRECATED METHODS *********** //
+		/* *********** DEPRECATED METHODS *********** */
 
 		/**
 		 * Retrieve plugin data, given the plugin name.
 		 *
 		 * @since      2.2.0
-		 * @deprecated 2.5.0
+		 * @deprecated 2.5.0 use {@see TGM_Plugin_Activation::_get_plugin_data_from_name()} instead.
 		 * @see        TGM_Plugin_Activation::_get_plugin_data_from_name()
```

Comment:
Actually - the handbook says you do: https://make.wordpress.org/core/handbook/inline-documentation-standards/php-documentation-standards/#1-2-deprecated-functions